File: Leetcode_Python/789_Escape_The_Ghosts.py
# https://leetcode.com/problems/escape-the-ghosts/description/



class Solution:
    def escapeGhosts(self, ghosts: List[List[int]], target: List[int]) -> bool:        
        for value in ghosts:
            if abs(target[0]-value[0])+abs(target[1]-value[1])<=abs(target[0])+abs(target[1]):
                return False
    
        return True

File: Leetcode_Javascript/789_Escape_The_Ghosts.js
// https://leetcode.com/problems/escape-the-ghosts/description/

var escapeGhosts = function (ghosts, target) {

    for (let item of ghosts) {
        if ((Math.abs(item[0] - target[0]) + Math.abs(item[1] - target[1])) <= Math.abs(target[0]) + Math.abs(target[1])) {
            return false
        }
    }
    return true


};